Makefile

```diff
@@ -99,6 +99,10 @@ release-artifacts: set-image-controller
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
+# Generate code
+generate: controller-gen
```

Comment:
moving it here to be closer to the template project https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/Makefile 

 + this allows me to see the difference between manifests and generate target